lib/grpc/google/api/annotations.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015, Google Inc.
```

Comment:
what's the usage for this file?

Reply:
They are required for the proxy/API gateway functionality. we define the endpoints inform of annotations.

Reply:
if so, please put them in a folder called `webProxy`, inside `grpc`, which can also contain `GrpcWebProxy.ts`. All the proxy stuff should be a separate sub module